template/cordova/lib/AppxManifest.js

```diff
@@ -390,11 +390,6 @@ AppxManifest.prototype.getVisualElements = function () {
         },
         setForegroundText: function (color) {
             if (color) {
```

Comment:
Instead of `if (color) { visualElements.attrib.ForegroundText = color; }` would it be better to use `visualElements.attrib.ForegroundText = color || 'light'` so that ForegroundText would be reset as default `light` if we set `dark` via preference for example and then remove the preference from config.xml entirely?